guides/source/active_record_basics.md

```diff
@@ -115,7 +115,7 @@ to Active Record instances:
 * `created_at` - Automatically gets set to the current date and time when the
   record is first created.
 * `updated_at` - Automatically gets set to the current date and time whenever
-  the record is updated.
+  the record is saved.
```

Comment:
I think this should be "the record is created or updated" because simply calling `save!` on a record won't change the timestamp so I think "the record is saved" is also misleading.